tools/RunVersionController.ps1

```diff
@@ -151,11 +151,13 @@ function Update-ChangeLog
     ($Content + $ChangeLogContent) | Set-Content -Path $ChangeLogFile.FullName -Encoding UTF8
 }
 
-if (!(Test-Path "C:/Program Files/PowerShell/Modules/PowerShellGet"))
+$PowerShellGetModuleList = gmo -l PowerShellGet | where version -ge '2.2.1'
+if ($PowerShellGetModuleList.Count -ne 0)
 {
+    $PowerShellGetPath = $PowerShellGetModuleList.path.Split('PowerShellGet')[0]
```

Comment:
What if `$PowerShellGetModuleList` is empty?  I think we just want to use Install-Module, which will automatically add this to the path if not selected, rather than downloading to a particular path.

`Install-Module -Scope CurrentUser` will automatically place the module first in the search order.

Alternately, we could just start the script with

``` powershell
#Requires -Modules @{ModuleName="PowerShellGet"; ModuleVersion="2.2.1"}
```

This will throw at the start of execution if it cannot find/load a version of PowerSgellGet >= 2.2.1